src/V3Const.h

```diff
@@ -43,6 +43,7 @@ class V3Const final {
     // Only the current node and lower, with special SenTree optimization
     // Return new node that may have replaced nodep
     static AstNode* constifyExpensiveEdit(AstNode* nodep);
+    static AstNode* constifyNode(AstNode* nodep);
 };
```

Comment:
You should be using constifyEdit I believe.

Reply:
I've tried to use `constifyEdit` instead of `constifyNode`, but `constifyEdit` sets `pmode` to `PROC_V_NOWARN` and then, verilator fails with message:
```
Internal: Unexpected Call
```

I've changed function name to `constifyCppNode` to be more explicite, that I'm using `PROC_CPP` mode.